Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.11.2-alpine
+
+COPY . /go/src/github.com/rakyll/hey
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 \
```

Comment:
```suggestion
RUN go mod download
RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 \
```

Reply:
Was there something I missed? I've used the image successfully, though not with https.

Reply:
This is nicer with reproducibility because it leverages Docker's cache. Should give better rebuild times.